### ISSUE TITLE
Councilmatic isn't updating from Legistar

### ISSUE THREAD

**Issue by stevevance** [datamade/chi-councilmatic#214]
New ordinances were posted Chicago's Legistar website on September 6, and they haven't been imported to Councilmatic. 

Sample ordinance that's not on Councilmatic yet: [O2017-6460](https://chicago.legistar.com/LegislationDetail.aspx?ID=3146563&GUID=B27C5C51-4763-4C72-A0F8-B2CBBA85C4C3&Options=Advanced&Search=)

**Comment by fgregg** [comment]
@reginafcompton can you take a look at this

**Comment by reginafcompton** [comment]
@fgregg - of course

**Comment by stevevance** [comment]
[Six zoning reclassification ordinances](https://chicago.councilmatic.org/search/?selected_facets=topics_exact%3AZoning%20Reclassification&_=1505144882132) have come in to Councilmatic today. There should be ~50 new zoning ordinances.

**Comment by reginafcompton** [comment]
@stevevance - we have a fix for this forthcoming. I'll send you an update, today!

**Comment by reginafcompton** [comment]
@stevevance: Legistar has a couple bills with duplicate actions, which have caused the scraper to throw errors (https://chicago.legistar.com/LegislationDetail.aspx?ID=3110435&GUID=5C6F4710-8B6F-487B-BCEF-BF8A56A3F030&Options=Advanced&Search=)

We are going to skip over those bills in our scrape, for the time being. I'll let you know when we have a successful scrape and import that data to Councilmatic.

**Comment by reginafcompton** [comment]
@stevevance - we've scraped the bills from September 6 (among others), which should be viewable on Councilmatic now. This recent scrape excludes the two bills with duplicate action items. 

Closing issue, but let us know if you see something missing.

**Comment by stevevance** [comment]
@reginafcompton You said there were two bills and linked to A2017-78 – what was the second bill?

**Comment by reginafcompton** [comment]
Yes, the other one: https://chicago.legistar.com/LegislationDetail.aspx?ID=3110438&GUID=53CF438E-7246-4785-B351-2417AFBE6C6B

**Comment by stevevance** [comment]
Thank you!

**Comment by stevevance** [comment]
@reginafcompton I found another example of Councilmatic not getting the latest data from Legistar: 

Legistar says [this ordinance](https://chicago.legistar.com/LegislationDetail.aspx?ID=3090440&GUID=E141B5D3-839A-4B1C-9F94-624D68E7BCDE) "passed in committee", but Councilmatic [doesn't have](https://chicago.councilmatic.org/legislation/o-2017-4818/) the "passed in committee" step and stops at "amended in committee".

**Comment by reginafcompton** [comment]
@stevevance - thanks, I'll reopen this issue.

**Comment by reginafcompton** [comment]
Legistar no longer contains the "passed in committee" action: https://chicago.legistar.com/LegislationDetail.aspx?ID=3090440&GUID=E141B5D3-839A-4B1C-9F94-624D68E7BCDE

It looks like Chicago Councilmatic has the correct information. Does this sound accurate @stevevance?

**Comment by stevevance** [comment]
I did a little more digging, and I looked at other ordinances, and it appears there's another action and I may have confused the two actions. 

1. The current status of `O2017-4818` is `Amended in Committee (pass)`, and is a committee-level action. 
2. The current status of `O2017-5512` is `Recommended to Pass (pass)`, and was also a committee-level action. 

I believe that action 1 has the same outcome as action 2: Both passed out of committee and on to City Council. The difference is that action 1 was amended before being passed and action 2 was not amended (or it could have been amended, but in a prior meeting and that ordinance would have a previous action).

I think that Councilmatic has the correct information for the ordinance `O2017-4818` I am concerned about, and that I've clarified my understanding of the possible actions.
